pkg/roll/run_update.go

```diff
@@ -399,7 +399,8 @@ func (u *update) shouldRollAfterDelay(newFields rcf.RC) (int, error) {
 }
 
 func (u *update) rollAlgorithmParams(oldHealth, newHealth rcNodeCounts) (int, int, int, int) {
```

Comment:
nit: name the return parameters so it's more obvious what is happening now that there is some logic here